views/layouts/main.handlebars

```diff
@@ -5,36 +5,47 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>MVC Patient Records</title>
-  <link rel="stylesheet" href="/stylesheets/style.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+  <link rel="stylesheet" href="/css/style.css">
 </head>
 
-<body>
-  <div class="wrapper">
+<body class="bg-dark text-white">
+  {{#if logged_in}}
```

Comment:
Not sure we need the if statements here -- I don't believe the user will be able to see the page _unless_ they're logged in